src/pages/color.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { Row, Col } from 'antd';
+import tinycolor from 'tinycolor2';
+
+const Color = () => {
+  const colors = new Array(10).fill(1);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['magic numbers' are not allowed: 10](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)